src/styles/auth-module.scss

```diff
@@ -160,6 +160,52 @@
         }
       }
     }
+
+    //Register
+    .register-view-form-wrapper{
+      margin: 0 auto;
+      @include auth-form-wrapper;
+      .heading{
+        padding-top: $default-padding*2;
+        padding-bottom: $default-padding*3;
+        @include screen-heading;
+      }
+      .ui.form{
+        .btn-wrapper{
+          margin-top: $default-margin*4;
+          margin-bottom: $default-margin*2;
+          @include align-item(flex, space-between, center, row);
+          .app-btn{
+            @include app-btn(100px, 50px, $theme-color, $theme-color);
+          }
+        }
+        .register-footer-wrapper {
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+        }
+      }
+      .register-success-form {
+        .sucess-msg, .error-msg {
+          font-size: 26px;
```

Comment:
Make the use of already created variables for fonts, padding, margin, etc. Above like would look something like `font-size: $default-font-size*2+6`